tech.json

```diff
@@ -2552,6 +2552,63 @@
                 "This tech should not be used directly; instead use a helper,",
                 "in order to correctly account for the amount of Crystal Flash refill, which depends on the environment."
               ]
+            },
+            {
+              "name": "canXModeBlueSuit",
```

Comment:
I think you should erase `canSuperJump` and apply it's ID here `160`
The only place it was used was in climb supers room, and it was to fall through the Speed blocks.

XMode__Suit techs are both SpeedBooster and XRay techs so just gotta pick one and where it is in SpeedBooster seems better.

Unless you have plans for upward momentum conservation that would fit a different structure better.